src/index.ts

```diff
@@ -44,23 +45,118 @@ export default class WorkspaceClient {
         return new RemoteAPI(resources);
     }
 
-    public static getJsonRpcApi(entryPoint: string): IWorkspaceMasterApi {
-        const transport = new WebSocketClient();
-        return new WorkspaceMasterApi(transport, entryPoint);
-    }
-
     private static createAxiosInstance(config: IRestAPIConfig): AxiosInstance {
-        if (config.ssCrtPath && this.isItNode() && fs.existsSync(config.ssCrtPath)) {
-            const agent = new https.Agent({
-                ca: fs.readFileSync(config.ssCrtPath)
-            });
-            return axios.create({httpsAgent: agent});
+        if (!this.isItNode()) {
+            this.addLogInterceptorsIfEnabled(axios);
```

Comment:
Not sure this makes sense to add interceptor in the browser case, since the `process` object will not be available ?

Reply:
I removed the reading of the env  variable instead I added a new property to config, so the logging can be controlled by the API